### PR TITLE
wcs: use jbuilder to build

### PR DESCRIPTION
**Pull request by avsm**
dune seems to result in a missing artefact error
spotted in revdeps for #15562

### CODE REVIEW

File: packages/wcs-api/wcs-api.2017-05-26.01/opam [hunk 1]
```diff
@@ -23,6 +23,7 @@ depends: [
   "wcs-lib" {= "2017-05-26.01"}
   "lwt_ssl"
   "cohttp-lwt-unix" {>= "0.20.0" & < "1.0"}
+  "jbuilder" {<="1.0+beta20.2"}
```
Comment (kit-ty-kate) [inline packages/wcs-api/wcs-api.2017-05-26.01/opam]:
```suggestion
  "jbuilder"
```
I'm not sure why add such a constraint. wcs seems to be able to build with any version of jbuilder (including transition): http://check.ocamllabs.io/log/1576115449-7b7ade392f48458c5399b9d5420146ef1b8605d2/4.02/good/wcs-lib.2017-05-26.02

File: packages/wcs-api/wcs-api.2017-05-26.02/opam [hunk 2]
```diff
@@ -23,6 +23,7 @@ depends: [
   "wcs-lib" {= "2017-05-26.02"}
   "lwt_ssl" | "tls"
   "cohttp-lwt-unix" {< "1.0.0"}
+  "jbuilder" {<="1.0+beta20.2"}
```
Comment (kit-ty-kate) [inline packages/wcs-api/wcs-api.2017-05-26.02/opam]:
```suggestion
  "jbuilder"
```

File: packages/wcs-api/wcs-api.2017-05-26.03/opam [hunk 3]
```diff
@@ -22,6 +22,7 @@ depends: [
   "ocaml" {>= "4.03.0"}
   "wcs-lib" {= "2017-05-26.03"}
   "cohttp-lwt-unix" {>= "1.0.0"}
+  "jbuilder" {<="1.0+beta20.2"}
```
Comment (kit-ty-kate) [inline packages/wcs-api/wcs-api.2017-05-26.03/opam]:
```suggestion
  "jbuilder"
```

File: packages/wcs-api/wcs-api.2017-05-26.04/opam [hunk 4]
```diff
@@ -22,6 +22,7 @@ depends: [
   "ocaml" {>= "4.03.0"}
   "wcs-lib" {= "2017-05-26.04"}
   "cohttp-lwt-unix" {>= "1.0.0"}
+  "jbuilder" {<="1.0+beta20.2"}
```
Comment (kit-ty-kate) [inline packages/wcs-api/wcs-api.2017-05-26.04/opam]:
```suggestion
  "jbuilder"
```

**Comment by avsm** [comment]
oddly though, it fails in cohttp revdeps with some version of dune. This PR is an improvement as it adds a missing dependency, so we can figure out the other problem separately

**Pull request merged**